sdk/src/main/java/com/oursky/authgear/latte/fragment/LatteFragment.kt

```diff
@@ -96,11 +192,16 @@ internal class LatteFragment() : Fragment() {
             if (fragment.webView.canGoBack()) {
                 fragment.webView.goBack()
             } else {
-                fragment.webView.completion?.invoke(fragment.webView, Result.failure(CancelException()))
+                fragment.broadcastCompleteIntent(Result.failure(CancelException()))
             }
         }
     }
 
+    override fun onAttach(context: Context) {
+        super.onAttach(context)
+        this.context = context
```

Comment:
Instead of storing `context`, use `getActivity` to get an instance of Activity, which is a subclass of Context.